src/sbt-test/lagom-conductr-bundle/lagom-java-service-execution/lagom-service-impl/src/main/java/impl/FooServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package impl;
+
+import akka.NotUsed;
+import com.lightbend.lagom.javadsl.api.ServiceCall;
+import java.util.concurrent.CompletableFuture;
+import javax.inject.Inject;
+import api.FooService;
+
+import akka.stream.javadsl.Source;
+
+public class FooServiceImpl implements FooService {
+
+  @Override
+  public ServiceCall<NotUsed, String> foo() {
+    return request -> CompletableFuture.completedFuture("foo-response-hardcoded in FooServiceImpl");
+  }
```

Comment:
Return a harcoded string used on the test to assert the response is the expected.